one_fm/purchase/doctype/quotation_comparison_sheet/quotation_comparison_sheet.js

```diff
@@ -139,3 +213,353 @@ var set_quotation_item_details = function(frm, item, quotation) {
 	qtn_item.rate = item.rate
 	qtn_item.amount = item.amount
 };
+
+
+
+// SET BUTTONS FOR QUOTATION COMPARISON
+let set_custom_buttons = (frm)=>{
+	if(![2,1].includes(frm.doc.docstatus)){
+		// Custom buttons in groups
+		frm.add_custom_button('Best Rate from One Supplier', () => {
+			best_price_same_supplier(frm);
+		}, 'Analyse');
+
+		// best_price_many_suppliers
+		frm.add_custom_button('Best Rate from Many Supplier', () => {
+			best_price_many_supplier(frm);
+		}, 'Analyse');
+
+		// best_price_many_suppliers
+		frm.add_custom_button('Earliest Delivery', () => {
+			earliest_delivery(frm);
+		}, 'Analyse');
+
+		// best_price_many_suppliers
+		frm.add_custom_button('Custom', () => {
+			customer_filter(frm);
```

Comment:
customer_filter. I hope you set it for custom_filter

Reply:
Ah, a rename will be required